scripts/run-serialization-roundtrip-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+export PYTHONPATH="$BUILD_DIR/lib/python"
+REVNG="$BUILD_DIR/bin/revng"
```

Comment:
See `tests/abi/test.sh` and `tests/abi/CMakeLists.txt`. Set `PATH` through `add_test`.